lib/Dialect/Calyx/CalyxOps.cpp

```diff
@@ -182,9 +182,9 @@ LogicalResult calyx::verifyComponent(Operation *op) {
 
 LogicalResult calyx::verifyCell(Operation *op) {
   auto opParent = op->getParentOp();
```

Comment:
Do we want to make sure that if this is a sequential cell, it is not located in a combinational component? Not sure if there is benefit to do it here rather than iteratively in the ComponentInterface verifier.

Reply:
I would rather not entangle the CellInterface with specific implementations of the ComponentInterface. It might be the case that in the future there are other kinds of Components that we want to introduce and I would rather it be the job of the specific ComponentOp verifier to check these sorts of things rather than it be the job of the CellInterface verifier.